packages/app/src/vis-packs/core/heatmap/HeatmapToolbar.tsx

```diff
@@ -109,14 +107,8 @@ function HeatmapToolbar(props: Props) {
 
       <Separator />
 
-      {getExportURL && (
-        <ExportMenu
-          isSlice={isSlice}
-          entries={EXPORT_FORMATS.map((format) => ({
-            format,
-            url: getExportURL(format),
-          }))}
-        />
+      {exportEntries.length > 0 && (
```

Comment:
If an `ExportEntry` is in the array, it is now sure to be supported (cf. `useExportEntries`). This allows hiding the export menu when there are no export entries (instead of just disabling it).